### DISCUSSION TITLE
VSCode highlighting

### DISCUSSION THREAD

**Discussion by rigrig** [scandum/tintin#195] [General]
I'm giving [VSCodium](https://vscodium.com/) a try, but sadly it lacked highlighting for TinTin++ scripts, so I made [this extension](https://git.tubul.net/rigrig/tintin_script). Maybe it'll come in handy for someone else. It's just some basic highlighting, but better than nothing.
(I'm not planning on jumping through all the hoops to publish it, so you need to install the `.vsix` file manually)

@scandum: would it be ok for me to use the [TinTin++ icon](https://github.com/scandum/tintin/blob/master/tintin-128.png) for it?

**Comment by scandum** [comment]
Sure, go ahead.

**Reply by rigrig** [reply]
Thanks!
